packages/sdk/src/actions/redeemSharesInKind.test.ts

```diff
@@ -0,0 +1,57 @@
+import { expect, test } from "vitest";
+import { toWei } from "../utils/conversion.js";
+import { publicClient, sendTestTransaction, testActions } from "../../tests/client.js";
+import { ALICE, WETH } from "../../tests/constants.js";
+import { simulateRedeemSharesInKind } from "./redeemSharesInKind.js";
+import { getBalanceOf } from "../../tests/actions/getBalanceOf.js";
+
+test("Redeem shares in kind should work correctly", async () => {
+  const { comptrollerProxy, vaultProxy } = await testActions.createTestVault({
+    vaultOwner: ALICE,
+    denominationAsset: WETH,
+  });
+
+  const balanceBeforeDeposit = await getBalanceOf({
+    token: vaultProxy,
+    account: ALICE,
+  });
+
+  const depositAmount = toWei(250);
+
+  const sharesBought = await testActions.buyShares({
+    comptrollerProxy,
+    sharesBuyer: ALICE,
+    investmentAmount: depositAmount,
+  });
+
+  expect(sharesBought).toBe(depositAmount);
```

Comment:
You are testing redemption here. So you don't need to assert the deposit outcome. Just the outcome of the withdrawal.